contracts/CO30_Matias_JokerAndTheThief.tem_suursaariv/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,9 @@
+<font color='#E3D310'>PRIMARY</font>
+1- Recover the stolen weapon shipments.
+Consisting of:
+-6 boxes of basic weapons
+-2 boxes of special weapons
+-3 boxes of launchers
+-4 boxes of grenades
+-3 boxes of explosvies
+-2 boxes of support items
```

Comment:
Wrong formatting. Leave a space before and after the dash (-), and don't number them the dashes, that's what the primary and secondary objective categories are for. The message will also not apply proper formatting once it is posted in the Discord's #schedule channel. Below is an example.

PRIMARY
Recover the stolen weapon shipments.
- 6 boxes of basic weapons
- 2 boxes of special weapons
- 3 boxes of launchers
- 4 boxes of grenades
- 3 boxes of explosvies
- 2 boxes of support items

Reply:
Numbering the top dash is unnecessary in this case, but I could have more than one Primary

Does the dot thing get applied when leaving a space before and after?

Reply:
yeah, it needs to be:
```
- Dash (space) Text
- - Dash (space) dash (space) subText
```